src/api/java/mekanism/api/datamaps/chemical/attribute/HeatedCoolant.java

```diff
@@ -59,7 +59,7 @@ public HeatedCoolant(Holder<Chemical> otherVariant, double thermalEnthalpy) {
     }
 
     public HeatedCoolant {
-        IChemicalCoolant.validateCoolantParams(otherVariant, thermalEnthalpy, conductivity);
+        IChemicalCoolant.validateCoolantParams(otherChemical(), thermalEnthalpy, conductivity);
```

Comment:
not this one. `validateCoolantParams` should take the wildcard and validate both kinds